pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -804,7 +825,7 @@ literals (via @racket[quote-syntax]) or variable references (via
 the syntactic specification of a @tech{cross-phase persistent} module
 declaration.
 
-A documented module should be assumed non-@tech{cross-phase persistent} unless it
+A documented module should be assumed non--@tech{cross-phase persistent} unless it
```

Comment:
pretty sure `-` gives the correct dash here

Reply:
I wanted an en-dash, which two hyphens produces.

Reply:
oh, and that's because "cross-phase" has a dash, ok